cypress/integration/plan.visit.spec.js

```diff
@@ -80,12 +85,13 @@ describe('Plan "huisbezoek"', () => {
     })
 
     it("Request is successfully processed", () => {
-      cy.wait(2000)
-      cy.get("h1").contains("Zaakdetails")
-      cy.wait(1000)
-      cy.get("h2").contains("Zaakhistorie")
-      cy.wait(1000)
-      cy.get("span").contains("Bezoek ingepland ")
+      const url = `${Cypress.env("baseUrlAcc")}cases/*/events/`
+      cy.intercept(url).as('getEvents')
+      cy.wait('@getEvents').then(() => {
+        cy.get("h1").contains("Zaakdetails")
+        cy.get("h2").contains("Zaakhistorie")
+        cy.get("span").contains("Bezoek ingepland ")
```

Comment:
This is ok, but you could also check for the item to be open or not and if the attribute 'title' is properly set. I test it like this in `addCase`:
cy.get('button[title="Melding "]')
      .should("have.attr", "aria-expanded", "true")
      .contains("Melding")
    cy.get('button[title="Aanleiding "]')
      .should("have.attr", "aria-expanded", "false")
      .contains("Aanleiding")`
`